upload_controller_service/api/main.py

```diff
@@ -56,3 +57,30 @@ async def get_presigned_post(
         ) from file_info_not_found_error
 
     return {"presigned_post": url}
+
+
+@app.get("/confirm_upload/{file_id}", summary="confirm_upload")
+async def confirm_upload(
+    file_id: str,
+    config: Config = Depends(get_config),
+):
+    """
+    Requesting a confirmation of the upload of a specific file using the file id.
+    Returns 200, if the file exists in the inbox, 404 if not
+    """
+
+    # call core functionality
+    try:
+        check_uploaded_file(file_id=file_id, config=config)
+    except FileInfoNotFoundError as file_info_not_found_error:
+        raise HTTPException(
+            status_code=400,
+            detail=f"The submitted file_id {file_id} does not exist.",
+        ) from file_info_not_found_error
+    except ObjectNotFoundError as object_not_found_error:
+        raise HTTPException(
+            status_code=404,
+            detail=f"The file with the file_id {file_id} does not exist.",
+        ) from object_not_found_error
+
+    return True
```

Comment:
Returning a 204 might be a bit better here. Something like:
```suggestion
@app.get("/confirm_upload/{file_id}", summary="confirm_upload", status_code=HTTP_204_NO_CONTENT)
async def confirm_upload(
    file_id: str,
    config: Config = Depends(get_config),
):
    """
    Requesting a confirmation of the upload of a specific file using the file id.
    Returns 204, if the file exists in the inbox, 404 if not
    """

    # call core functionality
    try:
        check_uploaded_file(file_id=file_id, config=config)
    except FileInfoNotFoundError as file_info_not_found_error:
        raise HTTPException(
            status_code=400,
            detail=f"The submitted file_id {file_id} does not exist.",
        ) from file_info_not_found_error
    except ObjectNotFoundError as object_not_found_error:
        raise HTTPException(
            status_code=404,
            detail=f"The file with the file_id {file_id} does not exist.",
        ) from object_not_found_error

    return Response(status_code=HTTP_204_NO_CONTENT)
```